Exercise_8/src/content.js

```diff
@@ -0,0 +1,6 @@
+/* global window:true */
+const setContent = (data) => {
+  window.document.getElementById('content').innerHTML = data;
```

Comment:
You still don't have this variable cached. Yes, it decreases performance by 0.001% but it is a good practice to cache DOM elements if they are reusable.